test cases/common/227 fs module/meson.build

```diff
@@ -18,7 +18,9 @@ assert(not fs.exists('nonexisting'), 'Nonexisting file was found.')
 is_git_checkout = fs.exists('../../../.git')
```

Comment:
I think we can get rid of `is_git_checkout` entirely from this file now that the symlink is generated at configure time.

Reply:
Thanks I removed that now unneeded code.